tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "declaration": true,
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es2020",
```

Comment:
This is necessary to use `Promise.allSettled()`. It doesn't seem to change any of the generated code in shiny.js. I believe it's because this is just the TypeScript config, and the actual compilation/transpilation is being done by something else.

The `"moduleResolution": "node"` was necessary after switching to `es2020`.